Applications/WebPortal/WebApp/tests/profile_page_tester/source/index.tsx

```diff
@@ -76,25 +77,24 @@ class TestApp extends React.Component<Properties, State> {
   }
 
   public componentDidMount() {
-    this.state.identity.photoId = TestApp.SOME_IMAGE;
-    this.state.identity.firstName = 'Frodo';
-    this.state.identity.lastName = 'Baggins';
-    this.state.identity.lastLoginTime = new Beam.DateTime(
+    const tempIdentity =  new Nexus.AccountIdentity();
```

Comment:
Remove extra space before ```new```.

Rename ```tempIdentity``` to ```identity```. I'd also recommend initializing it via ```this.state.identity.clone()```.